src/main/kotlin/dev/stalla/util/AnyExtensions.kt

```diff
@@ -17,16 +17,16 @@ internal fun allNotNull(
 }
 
 /** Check if at least one argument element is not null. */
-@InternalApi
+@InternalAPI
 internal fun anyNotNull(vararg elements: Any?): Boolean = elements.any { p -> p != null }
 
 /** Check if all argument elements are null. */
-@InternalApi
+@InternalAPI
 internal fun allNull(
     a: Any?,
-    b: Any? = Unit,
-    c: Any? = Unit,
-    d: Any? = Unit
+    b: Any? = null,
```

Comment:
Fairly important fix ehehe. ~Maybe we should test these utils in the future.~ Just noticed you added those, great!